source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2140,31 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, AIBridgeChannel_BridgeOffsetNullingCal_ReturnsError)
+{
+  const auto AI_MIN = 0.0;
+  const auto AI_MAX = 1.0;
+
+  EXPECT_THROW_DRIVER_ERROR({
+    auto status = create_ai_bridge_chan(AI_MIN, AI_MAX);
+  }, MEASUREMENT_TYPE_NOT_SUPPORTED_BY_SELECTED_PHYSICAL_CHANNEL_ERROR);
+}
+
+TEST_F(NiDAQmxDriverApiTests, AIThrmcplChannel_ThrmcplLeadOffsetNullingCal_ReturnsError)
```

Comment:
Please add a brief comment explaining why this is an error test (e.g. // X Series doesn't support thermocouple offset nulling)